assets/users/actions.ts

```diff
@@ -157,8 +157,9 @@ export function postUser() {
                     notify.success(gettext('User created successfully'));
                 }
                 dispatch(fetchUsers());
-            })
-            .catch((error: any) => errorHandler(error, dispatch, setError));
+            }, (error) => {
+                dispatch(setError(error));
```

Comment:
Rather than handling this case different than other errorHandlers, what if we were to improve the `errorHandler` function to check if the `error` is a request response or not, then dispatch `setError`.

Something like:
```typescript
(error.response?.json == null ?
    Promise.resolve(error) :
    error.response.json()
).then((data) => {
    dispatch(setError(data));
})
```

And continue to use `.catch` here rather than the rejected promise callback?

Reply:
tried to make it a bit more explicit